cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/CookieInterceptorBase.java

```diff
@@ -211,6 +222,15 @@ public boolean call(HttpConnection connection) throws IOException {
                         if (sessionHasStarted(connection.responseAsInputStream())) {
                             return storeCookiesFromResponse(connection.getConnection());
                         } else {
+                            // If the session did not start, consume the error stream
```

Comment:
Should probably include "to avoid leaking connections"

Reply:
Fixed in 86353f01a4170131ae71f2debd75069b0e4ead2d